docs/content/guides/developer/getting-started/data-serving.mdx

```diff
@@ -0,0 +1,115 @@
+---
+title: Access Sui Data
+description: Overview of the types of data access mechanisms available in Sui
+---
+
+You can access Sui network data like [Transactions](/concepts/transactions.mdx), [Checkpoints](/concepts/cryptography/system/checkpoint-verification.mdx), [Objects](/concepts/object-model.mdx), [Events](/guides/developer/sui-101/using-events.mdx) and more through the available interfaces, in order to use in your application workflows, or to analyze the network behavior across applications or protocols of interest, or to perform audits on parts or whole of the network.
+
+This document outlines the interfaces that are currently available to access the Sui network data, along with a overview of how that's gradually evolving for a great deveoper experience.
+
+## Current data access interfaces
+
+You can use either of the following mechanisms to access Sui network data currently:
+
+- Directly connect to [JSON-RPC](/references/sui-api.mdx) hosted by a Sui [full node](guides/operator/sui-full-node.mdx) where the [Mainnet](https://fullnode.mainnet.sui.io:443) or [Testnet](https://fullnode.testnet.sui.io:443) or [Devnet](https://fullnode.devnet.sui.io:443) load balancer URLs abstract a few globally distributed full nodes, OR
```

Comment:
Seeing `fullnode` written `full node` always seems to bother me, even though i know `full node` is likely more grammatically correct. In code its always written as `fullnode` but 🤷🏼‍♂️

Reply:
Agree and I've never liked "Full node" (full is also cappied). I'm going to change that soon but for consistency, it has to stay that way for now.